instruqt/proxy/track_scripts/setup-isolated-host

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+set -euxo pipefail
+
+echo "Running track setup script - isolated-host"
+epoc_start=`date +%s`
+
+# Wait for the Instruqt host bootstrap to finish.
+until [ -f /opt/instruqt/bootstrap/host-bootstrap-completed ]
+do
+    sleep 1
+done
+
+
+# Determine OS platform
+CENTOS=false
+UBUNTU=false
+if [ -f /usr/bin/python ] ; then
+  if python -mplatform |grep -i 'centos'; then CENTOS=true; fi
+fi
+if uname -a | grep -i Ubuntu ; then UBUNTU=true; fi
+
+
+# ubuntu
+if $UBUNTU ; then
+
+  NET_INTERFACE=ens4
+
+  # install some useful packages
+  apt-get update
+  apt-get --fix-broken install
+  apt-get install -y mlocate bind9utils net-tools jq
+
+  # enable proxy for apt (after setup package installs as proxy not available yet)
+  LOCAL_PROXY_IP=`host proxy-host | grep has | awk '{print $4}'`
+  cat <<EOF> /etc/apt/apt.conf.d/proxy.conf
+Acquire::http::Proxy "http://${LOCAL_PROXY_IP}:3128/";
+Acquire::https::Proxy "http://${LOCAL_PROXY_IP}:3128/";
+EOF
+fi
+
+# centos
+if $CENTOS ; then
+
+  NET_INTERFACE=eth0
+
+  # install some useful packages
+  yum install -y mlocate bind-utils
+
+  # Disable security services for smooth kurl install prompts
+  # note: stopping firewalld flushes iptables! so running before adding rules
+  systemctl stop firewalld
+  systemctl disable firewalld
+  setenforce 0
+
+  # enable proxy for yum (after setup package installs as proxy not available yet)
+  LOCAL_PROXY_IP=`host proxy-host | grep has | awk '{print $4}'`
+  echo "proxy=http://$LOCAL_PROXY_IP:3128" >> /etc/yum.conf
+fi
+
```

Comment:
Why do check for the OS (and have blocked you run for each) if you're specifying it when you select your OS image? Isn't one of these conditional blocks unreachable? I'm inclined to delete unreachable code to avoid confusion.

Reply:
Hmm yeah, at the time I was trying to make the code block portable between OS types, spawned out of originally both instances were centos but needed a different instance for the deploy host so used Josh's available instance which was ubuntu.  Would be cleaner to take this out now